### ISSUE TITLE
Improve the cosignatory display styles

### ISSUE THREAD

**Issue by OlegMakarenko** [nemgrouplimited/symbol-desktop-wallet#1377]
![image](https://user-images.githubusercontent.com/33131259/112765001-c0e5dc00-9013-11eb-8674-acf094142205.png)

**Comment by segfaultxavi** [comment]
I don't think the `=>` symbol is standard UI, users won't know what it means. In this case I think a colon (or even nothing at all) would suffice.

**Comment by OlegMakarenko** [comment]
@segfaultxavi I agree with you. The idea is to make it similar to the the mobile wallet view
![image](https://user-images.githubusercontent.com/33131259/113757109-e9a85880-971a-11eb-997b-9b714e5223c6.png)

**Comment by cryptoBeliever** [comment]
Related to https://github.com/symbol/desktop-wallet/issues/1453
